### PR TITLE
Add session delete capability fixes #175

### LINKED ISSUE #175: Admins should be able to cancel/delete a session
This appears to be a thing that has to happen fairly regularly, with people signing up to present and then their own schedule changes, and are unable.

The admin UI already has session create/update UI. Add the ability to remove a session.

### PR DESCRIPTION
**Pull request by aackerman**
Admins can delete any session or the owner can delete the session, and there's a confirmation.

Screen Shot 2019-07-10 at 7 49 07 PM

### CODE REVIEW

**Comment by aackerman** [comment]
@tonyc got any time to review these changes?

File: src/app/controllers/admin/sessions_controller.rb [hunk 1]
```diff
@@ -46,6 +46,12 @@ def create
     end
   end
 
+  def destroy
+    @session.destroy!
+    flash[:notice] = "Session has been deleted"
+    redirect_to '/admin/sessions'
```
Comment (tonyc) [inline src/app/controllers/admin/sessions_controller.rb]: Consider changing this to use the router helper, e.g. something like:

`redirect_to admin_sessions_path()`

I don't have the app running, so I can't check the exact name, but you should see it listed in the output of `rails routes`

File: src/app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -27,6 +27,12 @@ def update
     respond_with(@session)
   end
 
+  def destroy
+    @session.destroy!
+    flash[:notice] = "Your session has been deleted"
+    redirect_to '/'
```
Comment (tonyc) [inline src/app/controllers/sessions_controller.rb]: Likewise, consider using:

`redirect_to root_path()`

File: src/config/routes.rb [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
   get '/home' => 'pages#home', as: :home_page
 
-  resources :sessions, :only => [:index, :show, :new, :create, :update, :edit] do
+  resources :sessions, :only => [:index, :show, :new, :create, :update, :edit, :destroy] do
```
Comment (tonyc) [inline src/config/routes.rb]: Now that there's a `:destroy` action, you could remove all of the `:only` options:

`resources :sessions`

**Comment by tonyc** [comment]
Otherwise, if it works, :shipit: :+1:

**Comment by tonyc** [comment]
Also FWIW I'm unable to approve or deny PR's so someone else will have to do it :)

File: src/app/controllers/admin/sessions_controller.rb [hunk 4]
```diff
@@ -46,6 +46,12 @@ def create
     end
   end
 
+  def destroy
```
Comment (tonyc) [inline src/app/controllers/admin/sessions_controller.rb]: Dummy comment for review

**Comment by tonyc** [comment]
This is ready to merge, @caseyhelbling @pcantrell @jcoyne  ... not sure who else has merge access. Thanks!

**Comment by aackerman** [comment]
Those dynamic path fixes are easy, I can make the fixes

**Comment by pcantrell** [comment]
Trying your judgement, I can push the button!

**Pull request merged**